commands/end.js

```diff
@@ -9,10 +9,10 @@ module.exports = {
             message.channel.send(gameSession.scoreboard.getWinnerMessage());
             sendScoreboard(message, gameSession);
         }
-        else {
+        else if (gameSession.gameInSession()) {
```

Comment:
Just add a check for `gameSession.gameInSession()` at the beginning of the function and return instead